### ISSUE TITLE
--no-psuedo option does not seem to work

### ISSUE THREAD

**Issue by lparsons** [gpertea/gffread#43]
When specifying the `--no-pseduo` option, I see get pseudo features output. This differs from the behavior fo the version that was distributed with cufflinks.

`input.gtf`
```
19	snRNA	exon	223158	223261	.	-	.	 gene_id "ENSG00000222329"; transcript_id "ENST00000410397"; exon_number "1"; gene_name "U6"; gene_biotype "snRNA"; transcript_name "U6.795-201"; exon_id "ENSE00001807043";
19	unprocessed_pseudogene	exon	230084	230165	.	-	.	 gene_id "ENSG00000267600"; transcript_id "ENST00000587910"; exon_number "1"; gene_name "AC098474.1"; gene_biotype "pseudogene"; transcript_name "AC098474.1-001"; exon_id "ENSE00002913508";
19	unprocessed_pseudogene	exon	229640	229709	.	-	.	 gene_id "ENSG00000267600"; transcript_id "ENST00000587910"; exon_number "2"; gene_name "AC098474.1"; gene_biotype "pseudogene"; transcript_name "AC098474.1-001"; exon_id "ENSE00002957540";
19	pseudogene	exon	239145	239247	.	-	.	 gene_id "ENSG00000267305"; transcript_id "ENST00000588755"; exon_number "1"; gene_name "CTD-3113P16.7"; gene_biotype "pseudogene"; transcript_name "CTD-3113P16.7-001"; exon_id "ENSE00002958670";
19	pseudogene	exon	279495	280170	.	+	.	 gene_id "ENSG00000267447"; transcript_id "ENST00000589981"; exon_number "1"; gene_name "VN2R11P"; gene_biotype "pseudogene"; transcript_name "VN2R11P-001"; exon_id "ENSE00002783831";
19	protein_coding	exon	291285	291386	.	-	.	 gene_id "ENSG00000141934"; transcript_id "ENST00000269812"; exon_number "1"; gene_name "PPAP2C"; gene_biotype "protein_coding"; transcript_name "PPAP2C-001"; exon_id "ENSE00001234447";
```

Command:
```
gffread Homo_sapiens.GRCh37_19.71.gtf --no-pseudo     -T -o output.gtf
```

Output from old cufflinks `gffread`:
```
19	snRNA	exon	223158	223261	.	-	.	transcript_id "ENST00000410397"; gene_id "ENSG00000222329"; gene_name "U6"; gene_biotype "snRNA"; transcript_name "U6.795-201";
19	protein_coding	exon	281043	281537	.	-	.	transcript_id "ENST00000269812"; gene_id "ENSG00000141934"; gene_name "PPAP2C"; gene_biotype "protein_coding"; transcript_name "PPAP2C-001";
```

Output from new 0.11.4 `gffread`:
```
19	snRNA	transcript	223158	223261	.	-	.	transcript_id "ENST00000410397"; gene_id "ENSG00000222329"; gene_name "U6"; gene_biotype "snRNA"; transcript_name "U6.795-201";
19	snRNA	exon	223158	223261	.	-	.	transcript_id "ENST00000410397"; gene_id "ENSG00000222329"; gene_name "U6";
19	unprocessed_pseudogene	transcript	229640	230165	.	-	.	transcript_id "ENST00000587910"; gene_id "ENSG00000267600"; gene_name "AC098474.1"; gene_biotype "pseudogene"; transcript_name "AC098474.1-001";
19	unprocessed_pseudogene	exon	229640	229709	.	-	.	transcript_id "ENST00000587910"; gene_id "ENSG00000267600"; gene_name "AC098474.1";
19	unprocessed_pseudogene	exon	230084	230165	.	-	.	transcript_id "ENST00000587910"; gene_id "ENSG00000267600"; gene_name "AC098474.1";
19	pseudogene	transcript	239145	239247	.	-	.	transcript_id "ENST00000588755"; gene_id "ENSG00000267305"; gene_name "CTD-3113P16.7"; gene_biotype "pseudogene"; transcript_name "CTD-3113P16.7-001";
19	pseudogene	exon	239145	239247	.	-	.	transcript_id "ENST00000588755"; gene_id "ENSG00000267305"; gene_name "CTD-3113P16.7";
19	pseudogene	transcript	279495	280170	.	+	.	transcript_id "ENST00000589981"; gene_id "ENSG00000267447"; gene_name "VN2R11P"; gene_biotype "pseudogene"; transcript_name "VN2R11P-001";
19	pseudogene	exon	279495	280170	.	+	.	transcript_id "ENST00000589981"; gene_id "ENSG00000267447"; gene_name "VN2R11P";
19	protein_coding	transcript	281043	291386	.	-	.	transcript_id "ENST00000269812"; gene_id "ENSG00000141934"; gene_name "PPAP2C"; gene_biotype "protein_coding"; transcript_name "PPAP2C-001"; protein_id "ENSP00000269812";
```

**Comment by gpertea** [comment]
Thank you for reporting this, it looks like I made this filter much more stringent (probably too specific) in the recent versions. I was trying to avoid false positives so I actually started to only look into a very limited set of attributes and other key strings (modeled after the NCBI annotation files). 

I seem to recall that at some point the older version was simply looking for the 'pseudo' keyword anywhere in the transcript line (either attributes or values) and that was surely a problem as some protein or mRNA descriptions may have "pseudo" in there but may not actually mean a pseudogene.

It looks like looking for "pseudogene" keyword (and for constructs like `*type=pseudogene` or `*type= *_pseudogene`) should catch most of the "real" cases, so I am going to try to fix this now using this kind of matches (suggestions are welcome). I am still going to ignore the "pseudogene" keyword if it's just in the "track" (2nd) column, even though in this example seems to give it away pretty clearly..  I think I should count on GTF/GFF attributes only.
